tutorials/polkadot-sdk/parachains/build-custom-pallet/.pages

```diff
@@ -0,0 +1,4 @@
+title: Build Custom Pallet
+nav:
+  - index.md
+  - 'Build Pallet': build-pallet.md
```

Comment:
Just curious, what other tutorials are going to be in this section?

Reply:
Pallet Unit Testing and Pallet Benchmarking